scripts/build-ancillary-repository.sh

```diff
@@ -94,7 +94,7 @@ function build_delphix_java8_debs() {
 	local pkg_directory="$1"
 
 	local url="http://artifactory.delphix.com/artifactory"
-	local tarfile="jdk-8u131-linux-x64.tar.gz"
+	local tarfile="jdk-8u171-linux-x64.tar.gz"
 	local jcefile="jce_policy-8.zip"
```

Comment:
Java 8 Update 171 no longer requires a separate JCE Policy, as far as I know. (See [this review](http://reviews.delphix.com/r/40951/) for details, or ask Brandon.) I think this should therefore be deleted (along with the `--jce-policy` argument to `make-jpkg` below).